omod/src/main/resources/config.xml

```diff
@@ -66,6 +66,9 @@
         <require_module>
             org.openmrs.module.idgen
         </require_module>
+		<require_module>
+			org.openmrs.module.legacyui
```

Comment:
Why not aware of instead of required? Do you intentionally want the module not to run on lower platform versions?